include/world_builder/bounding_box.h

```diff
@@ -93,7 +93,7 @@ namespace WorldBuilder
    *
    * @note The majority of this class is copied from the deal.II library.
    */
-  template <int spacedim>
+  template <unsigned int spacedim>
```

Comment:
is this unrelated or necessary? Note that we use ``int`` in ASPECT and deal.II.

Reply:
It is the easiest way to get rid of the new warnings and logically the most consistent. I don't think it will impact ASPECT in any way, since the types are not shared.